tests/e2e/deployment-test-cleanup.sh

```diff
@@ -17,9 +17,9 @@ source ./tests/e2e/lib-tests.sh
 
 helm -n dynatrace ls --all --short | grep dynatrace-gcp-monitor | xargs -L1 helm -n dynatrace uninstall --timeout 10m
 
+gcloud logging sinks delete "${LOG_ROUTER}"
```

Comment:
This is only to avoid some errors being logged when the sink was not finding the topic, between deletions.